data/mods/Xedra_Evolved/monsters/inventor.json

```
@@ -0,0 +1,131 @@
+[
+  {
+    "id": "mon_sniperbot",
+    "type": "MONSTER",
+    "name": { "str": "sniper bot" },
+    "description": "Lorem ipsum.",
```

```suggestion
    "description": "Quadrupedal legs with a rifle barrel mounted across the back of this doglike robot, appear to grant a high level of stability in order to strike targets at long range.",
``` 

Just a start or idea.